components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/cache/APIAccessTokenCache.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License? I think All rights resolved should be removed. please check

Reply:
Resolved with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/0bf9bde5d8f18456b6d0304605a1e69596024900